src/Microsoft.Azure.ServiceBus/Extensions/MessageInterOpExtensions.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.Azure.ServiceBus.InteropExtensions
```

Comment:
>InteropExtensions [](start = 37, length = 17)

the choice was deliberate to name namespace as interop, after we got code review comments to say the extension namespace should be by role rather than the implementations... Can you please rename this back